app/models/template.rb

```diff
@@ -206,12 +206,16 @@ class Template < ApplicationRecord
   }
 
   # Retrieves unarchived templates whose title or org.name includes the term
-  # passed
+  # passed(We use search_term_orgs as alias for orgs to avoid issues with
+  # any orgs table join already present in loaded unarchived.)
   scope :search, lambda { |term|
-    unarchived.joins(:org)
-              .where('lower(templates.title) LIKE lower(:term) OR ' \
-                     'lower(orgs.name) LIKE lower(:term)',
-                     term: "%#{term}%")
+    unarchived
+      .joins(<<~SQL)
```

Comment:
Oh interesting... I copied the format from examples on line #137 and #150 in the same file. I tried making the change you suggested locally and it seems to mess up the formatting 🤔

I'm hoping that since the current format has been used in the past, has now passed rubocop checks and works..then maybe it can stay the same? Let me know what you think.

Reply:
oh weird ok. If it works then I'm happy. 

Typically with [heredoc](https://www.rubyguides.com/2018/11/ruby-heredoc/), anything after the `<<~SQL ... SQL` becomes part of the text. Maybe the `)` is required to close out the prior part of the SQL statement that ActiveRecord is generating behind the scenes.

Reply:
It definitely works, but John just mentioned that he would prefer the second suggestion you made which is to store the SQL part in a var and do `.joins(var)`,as that it more clear. So I might make those changes and push it again. Will keep you posted.

We should then change the format of the other two scopes as well I'm guessing, so that they are all of the same format (?)